### PR TITLE
CC package update

### PR DESCRIPTION
**Pull request by fhnaseer**
Updating code coverage package. Adding dependent dlls to nuget package

### CODE REVIEW

File: TestPlatform.sln [hunk 1]
```diff
@@ -934,18 +934,6 @@ Global
 		{62E9D32B-B989-43CF-81A2-B38B3367FCA3}.Release|x64.Build.0 = Release|Any CPU
 		{62E9D32B-B989-43CF-81A2-B38B3367FCA3}.Release|x86.ActiveCfg = Release|Any CPU
 		{62E9D32B-B989-43CF-81A2-B38B3367FCA3}.Release|x86.Build.0 = Release|Any CPU
-		{4DA57968-F547-4019-8381-03A218B6C385}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (Evangelink) [inline TestPlatform.sln]: Could we remove all un-needed entries?

Comment (fhnaseer) [reply TestPlatform.sln]: Removed,

**Comment by nohwnd** [comment]
Converted to actual draft PR.

**Review by Haplois — APPROVED** [review]

File: src/package/nuspec/Microsoft.CodeCoverage.nuspec [hunk 2]
```diff
@@ -36,6 +36,12 @@
     <file src="Microsoft.CodeCoverage\Microsoft.CodeCoverage.Core.dll" target="build\netstandard1.0\" />
     <file src="Microsoft.CodeCoverage\Microsoft.CodeCoverage.Interprocess.dll" target="build\netstandard1.0\" />
     <file src="Microsoft.CodeCoverage\Microsoft.CodeCoverage.Instrumentation.dll" target="build\netstandard1.0\" />
+    <file src="Microsoft.CodeCoverage\Microsoft.CodeCoverage.Telemetry.dll" target="build\netstandard1.0\" />
```
Comment (jakubch1) [inline src/package/nuspec/Microsoft.CodeCoverage.nuspec:39]: how big are those dlls? I'm wondering if it is good to add it also to .netstandard version of collector.

File: eng/Versions.props [hunk 3]
```diff
@@ -82,8 +82,12 @@
          JS etc.) is risky. This can break setup authoring and cause runtime failures in workloads
          where Rolsyn is not installed. -->
     <MicrosoftCodeAnalysisVersion>3.8.0-3.20427.2</MicrosoftCodeAnalysisVersion>
-    <MicrosoftInternalCodeCoverageVersion>17.4.3-beta.22374.1</MicrosoftInternalCodeCoverageVersion>
+    <MicrosoftInternalCodeCoverageVersion>17.4.3-beta.22375.3</MicrosoftInternalCodeCoverageVersion>
     <MicrosoftDiagnosticsNETCoreClientVersion>0.2.0-preview.21508.1</MicrosoftDiagnosticsNETCoreClientVersion>
     <MicrosoftSourceBuildIntermediatediagnosticsVersion>5.0.0-preview.21508.1</MicrosoftSourceBuildIntermediatediagnosticsVersion>
+    <MicrosoftVisualStudioTelemetryVersion>16.4.78</MicrosoftVisualStudioTelemetryVersion>
```
Comment (jakubch1) [inline eng/Versions.props:88]: what will happen if those versions do not match version in code coverage repo? Maybe it's better to include dlls in Microsoft.Internal.CodeCoverage. Then those version are not needed here. Let's discuss this tomorrow

File: src/package/nuspec/Microsoft.CodeCoverage.nuspec (hunk 2 above)
Comment (fhnaseer) [reply src/package/nuspec/Microsoft.CodeCoverage.nuspec:39]: 0.98MB combined

**Comment by Evangelink** [comment]
Could you also update the description of this PR when it's ready?

File: eng/Versions.props [hunk 4]
```diff
@@ -82,8 +82,11 @@
          JS etc.) is risky. This can break setup authoring and cause runtime failures in workloads
          where Rolsyn is not installed. -->
     <MicrosoftCodeAnalysisVersion>3.8.0-3.20427.2</MicrosoftCodeAnalysisVersion>
-    <MicrosoftInternalCodeCoverageVersion>17.4.3-beta.22374.1</MicrosoftInternalCodeCoverageVersion>
+    <MicrosoftInternalCodeCoverageVersion>17.4.4-beta.22376.6</MicrosoftInternalCodeCoverageVersion>
     <MicrosoftDiagnosticsNETCoreClientVersion>0.2.0-preview.21508.1</MicrosoftDiagnosticsNETCoreClientVersion>
     <MicrosoftSourceBuildIntermediatediagnosticsVersion>5.0.0-preview.21508.1</MicrosoftSourceBuildIntermediatediagnosticsVersion>
+    <MicrosoftVisualStudioTelemetryVersion>16.4.78</MicrosoftVisualStudioTelemetryVersion>
+    <MicrosoftVisualStudioRemoteControlVersion>16.3.44</MicrosoftVisualStudioRemoteControlVersion>
```
Comment (jakubch1) [inline eng/Versions.props:89]: this can be removed, ritght?

Comment (fhnaseer) [reply eng/Versions.props:89]: it is used in build.ps1 for other place (line 815). Moved version number here,

Comment (jakubch1) [reply eng/Versions.props:89]: so those packages were referenced by TP before telemetry changes?

Comment (jakubch1) [reply eng/Versions.props:89]: this is first commit => its ok

Comment (fhnaseer) [reply eng/Versions.props:89]: yes,

**Review by jakubch1 — APPROVED** [review]

**Pull request merged**